presto-main/src/main/java/com/facebook/presto/event/query/QueryMonitor.java

```diff
@@ -136,7 +136,7 @@ public void queryCompletedEvent(QueryInfo queryInfo)
             ImmutableList.Builder<QueryInputMetadata> inputs = ImmutableList.builder();
             for (Input input : queryInfo.getInputs()) {
                 inputs.add(new QueryInputMetadata(
-                        input.getConnectorId(),
+                        input.getConnectorId().toString(),
```

Comment:
Should these be `getCatalogName()`? The methods do the same thing, but it's not clear why the usages are mixed.